exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>"However, sometimes this might not be what we expected.".  That actually means that something has gone wrong, doesn't it. The tone of this sentence can make one think that this is still normal behaviour and the appearance could be deceptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Removed the sentence</p>